package.json

```diff
@@ -96,6 +96,8 @@
     "**/minimist": "^1.2.5",
     "**/node-jose/node-forge": "^0.10.0",
     "**/prismjs": "1.22.0",
+    "**/react-syntax-highlighter": "^15.3.1",
```

Comment:
Forcefully resolves:

* `@storybook/components > react-syntax-highlighter@12.2.1` to `^15.3.1`

See [`react-syntax-highlighter` changelog](https://github.com/react-syntax-highlighter/react-syntax-highlighter/blob/master/CHANGELOG.MD).